src/Capgemini.PowerApps.PackageDeployerTemplate/PackageTemplateBase.cs

```diff
@@ -15,6 +16,7 @@
     /// <summary>
     /// A base Package Deployer template class that provides additional configurable deployment functionality.
     /// </summary>
+    [Export(typeof(IImportExtensions))]
```

Comment:
What is this for?

Reply:
Good catch - should've been added to the mock template and not the base template.